server/cron_service.go

```diff
@@ -0,0 +1,137 @@
+package server
+
+import (
+	"os"
+	"os/signal"
+	"strconv"
+	"syscall"
+	"time"
+
+	stats "github.com/lyft/gostats"
+	"github.com/runatlantis/atlantis/server/events"
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/logging"
+)
+
+type CronService struct {
+	log logging.SimpleLogging
+
+	// jobs
+	garbageCollector CronDefinition
+}
+
+func NewCronService(
+	workingDirIterator events.WorkDirIterator,
+	statsScope stats.Scope,
+	log logging.SimpleLogging,
+) *CronService {
+	garbageCollector := &GarbageCollector{
+		workingDirIterator: workingDirIterator,
+		stats:              statsScope.Scope("crons.garbagecollector"),
+		log:                log,
+	}
+
+	garbageCollectorCron := CronDefinition{
+		Job: garbageCollector,
+
+		// 5 minutes should probably be the lowest to prevent GH rate limits
+		Period: 5 * time.Minute,
```

Comment:
This can probably run even less often. Like once every hour.

Reply:
Yeah probably better to be safe.

Reply:
I think I'll keep it at 5 for debugging purposes at the start and then change it later.